common/type.c

```diff
@@ -10,8 +10,12 @@ add_type(sqlite3 *db, const char *type)
 	sqlite3_stmt *stmt;
 	const char *unused;
 
+	if (type_exists(db, type))
```

Comment:
Brakets.

Reply:
Not according to style guide

Reply:
However I find this one of most common reasons of bugs in software. Including Apple safety bug.
http://www.slate.com/articles/technology/bitwise/2014/02/apple_security_bug_a_critical_flaw_was_extraordinarily_simple.html

Reply:
I prefer the simplicity of lack of brace, but if you really prefer we can use them

Reply:
Don't get me wrong, I also like the simplicity, but I also have seen too many problems because of that. So I vote for brackets always.

Reply:
Updated with brace

Reply:
No braces here.